Products/CMFPlone/RegistrationTool.py

```diff
@@ -226,8 +226,8 @@ def pasValidation(self, property, password):
                     err = error
                 else:
                     msgid = _(
-                        "${sentances}. ${sentance}",
-                        mapping={"sentances": err, "sentance": error},
+                        "${sentences}. ${sentence}",
+                        mapping={"sentences": err, "sentence": error},
```

Comment:
@erral `codespell` was run, and this may mean that similar changes are needed in `plone.app.locales`.  Currently this PR is for CMFPlone master, which since today is only for Plone 6.1.  But this PR should be back ported soon to new branch 6.0.x.

Reply:
OK. Some weeks ago I also modified manually the translations in such cases, so I would need to do it again when this is backported to 6.0.x.

Thanks for the heads up.